prettier/stylelint-config-prettier-scss
# Project Cleanup

* Add Node 18 tests
* Update node engines to LTS support
* Add volta config to ensure Node 18 used locally
* Regenerate lockfile with latest Node LTS
* Update eslint parser to use latest ecmaVersion
* Run prettier formatting
* Update contributor list
* Update naming/format for README/LICENSE
